src/packagedcode/nuget.py

```diff
@@ -124,7 +124,7 @@ def _parse_nuspec(location):
     """
     if not location.endswith('.nuspec'):
         return
-    with open(location) as loc:
+    with open(location , "rb") as loc:
```

Comment:
Can you elaborate why opening 'rb' is needed? Also please use single quotes.

Reply:
Read this https://github.com/martinblech/xmltodict/issues/77